frontend/vue/components/CodeExercise/ExerciseActionsBar.vue

```diff
@@ -98,6 +98,13 @@ export default defineComponent({
     @include bicolor-background($button-background-color-dark, $button-background-color);
     color: $button-text-color;
 
+    &_disabled {
+      background-color: $disabled-background-color;
+      background-image: none;
+      color: $text-color-lighter;
+      cursor: not-allowed;
```

Comment:
This is to match the cursor property in the disabled buttons in Carbon: https://carbondesignsystem.com/components/button/code/

It's also much better for the UX, since the unavailability of the interaction is clear on hover before clicking.